test/browser/element.vdiff.js

```diff
@@ -223,29 +232,31 @@ describe('element-different', () => {
 				fail = true;
 			}
 
-			expect(fail, 'current and golden images to be different').equal(!isGolden);
-
-			if (!isGolden) {
-				await executeServerCommand('vdiff-revert-golden-file', { testCategory: 'element-different', fileName: `${name}.png` });
-			}
+			expect(fail, 'current and golden images to be different').equal(expectFailures);
+			await executeServerCommand('vdiff-remove-test-files', { testCategory: 'element-different', fileName: `${name}.png` });
```

Comment:
We run this _after_ the `expect` so that if it fails (i.e. the images are unexpectedly the same), we don't clean up the files so a commit can happen and the report will be uploaded and use them.